utils/subpath.go

```diff
@@ -69,9 +69,13 @@ func UpdateAssetsSubpath(subpath string) error {
 	script := fmt.Sprintf("window.publicPath='%s'", newPath)
 	scriptHash := sha256.Sum256([]byte(script))
 
-	reCSP := regexp.MustCompile(`<meta http-equiv=Content-Security-Policy content="script-src 'self' cdn.segment.com/analytics.js/ 'unsafe-eval'([^"]*)">`)
+	reCSP := regexp.MustCompile(`<meta http-equiv="Content-Security-Policy" content="script-src 'self' cdn.segment.com/analytics.js/ 'unsafe-eval'([^"]*)">`)
```

Comment:
Assuming the previous implementation was meaningful, should the new quotes be optional? Or do we only apply `reCSP` to the inputs we control, below?

Reply:
We only apply `reCSP` to inputs we control, specifically the `root.html` file that Webpack generates as part of the build process. Its previous output didn't escape the `http-equiv` attribute's value, but the new version does.